Ah, sorry, should have pasted that in the first place:

Those were the errors I started with in 5.9.6:

```
      psutil/_psutil_linux.c: In function ‘ioprio_get’:
      psutil/_psutil_linux.c:73:12: error: implicit declaration of function ‘syscall’ [-Werror=implicit-function-declaration]
         73 |     return syscall(__NR_ioprio_get, which, who);
            |            ^~~~~~~
      psutil/_psutil_linux.c: In function ‘psutil_net_if_duplex_speed’:
      psutil/_psutil_linux.c:488:5: error: implicit declaration of function ‘close’; did you mean ‘clone’? [-Werror=implicit-function-declaration]
        488 |     close(sock);
            |     ^~~~~
            |     clone
```

Then I checked on the master branch and haven't noticed the error is a tad different:

```
      psutil/arch/linux/net.c: In function ‘psutil_net_if_duplex_speed’:
      psutil/arch/linux/net.c:112:5: error: implicit declaration of function ‘close’; did you mean ‘pclose’? [-Werror=implicit-function-declaration]
        112 |     close(sock);
            |     ^~~~~
            |     pclose
      cc1: some warnings being treated as errors
```

Will move the include to `psutil/arch/linux/net.c` and amend the commit message.

--- a/psutil/_psutil_sunos.c
+++ b/psutil/_psutil_sunos.c
@@ -51,6 +51,7 @@
 #include <arpa/inet.h>
 #include <net/if.h>
 #include <math.h> // fabs()
+#include <unistd.h>
 
 #include "_psutil_common.h"
 #include "_psutil_posix.h"

--- a/psutil/arch/linux/net.c
+++ b/psutil/arch/linux/net.c
@@ -10,6 +10,7 @@
 #include <sys/ioctl.h>
 #include <linux/if.h>
 #include <linux/version.h>
+#include <unistd.h>
 
 // see: https://github.com/giampaolo/psutil/issues/659
 #ifdef PSUTIL_ETHTOOL_MISSING_TYPES

--- a/psutil/arch/linux/proc.c
+++ b/psutil/arch/linux/proc.c
@@ -7,6 +7,7 @@
 #include <Python.h>
 #include <sys/syscall.h>
 #include <sched.h>
+#include <unistd.h>
 
 #include "proc.h"
 #include "../../_psutil_common.h"

--- a/psutil/arch/solaris/environ.c
+++ b/psutil/arch/solaris/environ.c
@@ -19,6 +19,7 @@
 #include <sys/procfs.h>
 #include <sys/stat.h>
 #include <fcntl.h>
+#include <unistd.h>
 
 #include "environ.h"
 

